### PR TITLE
Rich Text positioning

### PR DESCRIPTION
**Pull request by miguel-merlin**
## Description
The size of the button inside the `RichText` component spanned all the container. If fixed it by letting the user provide a positioning option in the `RichText` component. 
image

### CODE REVIEW

**Review by ShawnAviles — DISMISSED** [review]
LGTM 👍

**Comment by miguel-merlin** [comment]
Upon further testing the change. It seems that when right is selected, the component does not go all the way right. There is a huge padding. @ShawnAviles any ideas?

File: src/components/blocks/layout/RichText.tsx [hunk 1]
```diff
@@ -5,14 +5,21 @@ import {
...
 export const RichText = ({ data }: { data: PageBlocksRichText }) => {
-	const { richTextContent } = data
 	return (
-		<div className='max-w-[1360px] mx-auto p-10'>
-			<div className='flex flex-col grow justify-center prose max-w-none prose-headings:my-6 prose-p:my-2 p'>
-				<TinaMarkdown components={tinaMarkdownComponents} content={richTextContent} />
-			</div>
+		<div
+			className={cn(
+				'prose px-4 md:prose-h1:text-6xl prose-h1:text-4xl prose-headings:my-6 prose-p:my-2 max-w-7xl flex flex-col gap-4',
```
Comment (ShawnAviles) [inline src/components/blocks/layout/RichText.tsx]: just need to add `mx-auto` to fix the extra right padding. It is currently restricted due to the max-w-7xl but that's fine bc we don't want it to take up the whole page so we cap it there. The auto horiztonal margin `mx-auto` ensures that the usable maximum width of 7xl (i think ~1280px) is centered too. 

It was not centered without it, hence why it looked like there was too much right padding on the right layout as it was aligned on the left by default

**Review by ShawnAviles — APPROVED** [review]

File: src/components/blocks/layout/RichText.tsx (hunk 1 above)
Comment (miguel-merlin) [reply src/components/blocks/layout/RichText.tsx]: Thanks @ShawnAviles! I should really learn css layouts this winter break

**Pull request merged**